README.md

```diff
@@ -10,8 +12,13 @@ An elemental block to enable more control over the responsiveness of tables.
     }
 ],
 "require": {
-    "signify-nz/silverstripe-factory-elemental-tables": "dev-master"
+    "signify-nz/silverstripe-factory-elemental-tables": "^1"
 }
 ```
 
 ## Usage
+This module concentrates on semantically marking up an HTML table to avoid misuse and/or poor results from inserting a table in the WYSIWYG field and applying inline styles which overide all cascade specificity.
+
+Be aware that this module does not come with css. Settings that provide classes to trigger visual changes (including the responsive behaviour/transposition, Zebra rows, Border modifications etc.) only provide a framework for you to build your theme around.
```

Comment:
How can one determine the classes to implement, or what to change to add classes for this?